backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceAsync.java

```diff
@@ -116,30 +116,31 @@ public CompletableFuture<Set<Person>> savePatients(byte[] content, UUID facility
 
         // create new person with current organization, then add to new patients list
         Person newPatient =
-            new Person(
-                currentOrganization,
-                assignedFacility.orElse(null),
-                null, // lookupid
-                extractedData.getFirstName().getValue(),
-                extractedData.getMiddleName().getValue(),
-                extractedData.getLastName().getValue(),
-                extractedData.getSuffix().getValue(),
-                parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-                address,
-                country,
-                parsePersonRole(extractedData.getRole().getValue(), false),
-                extractedData.getEmail().getValue() == null
-                    ? Collections.emptyList()
-                    : List.of(extractedData.getEmail().getValue()),
-                convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-                convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-                null, // tribalAffiliation
-                convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-                parseYesNoUnk(extractedData.getResidentCongregateSetting().getValue()),
-                parseYesNoUnk(extractedData.getEmployedInHealthcare().getValue()),
-                null, // preferredLanguage
-                null // testResultDeliveryPreference
-                );
+            Person.builder()
+                .organization(currentOrganization)
+                .facility(assignedFacility.orElse(null))
+                .birthDate(parseUserShortDate(extractedData.getDateOfBirth().getValue()))
+                .address(address)
+                .country(country)
+                .role(parsePersonRole(extractedData.getRole().getValue(), false))
+                .emails(
+                    extractedData.getEmail().getValue() == null
+                        ? Collections.emptyList()
+                        : List.of(extractedData.getEmail().getValue()))
+                .race(convertRaceToDatabaseValue(extractedData.getRace().getValue()))
+                .ethnicity(convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()))
+                .gender(convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()))
+                .genderIdentity(extractedData.getGenderIdentity().getValue())
```

Comment:
Just so you know when adding `genderIdentity`, for example `TRANSMAN` will save but on the frontend it won't display because the value we are looking for is all lowercase.

Reply:
thanks for the catch! fixed and added a regression test